kategory-core/src/main/kotlin/kategory/data/SortedMapKW.kt

```diff
@@ -0,0 +1,77 @@
+package kategory
+
+import java.util.*
+
+@higherkind
+data class SortedMapKW<K: Comparable<K>, A>(val map: SortedMap<K, A>) : SortedMapKWKind<K, A>, SortedMap<K, A> by map {
```

Comment:
If we are going to follow here the `K, V` instead of `A, B` we should rename `A` for `V` so it denotes we refer to the value associated to a key otherwise use `A, B` which is the Kategory convention for applied types that are not in higher kind position.

Reply:
I didn't know `A, B` was the convention. I will change 👌